package.json

```diff
@@ -13,6 +13,7 @@
     "init:windows": "node --env-file=\".\\.env\" \".\\app\\(api)\\_utils\\mongodb\\dbInit.mjs\""
   },
   "dependencies": {
+    "dotenv": "^16.4.5",
```

Comment:
Is it possible to not have to change the package.json? I believe env files should be loaded automatically by next.js. Let me know if you have trouble getting it to work without dotenv